ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -130,6 +133,31 @@
           <code class="has-text-grey is-size-8">
             {{backend.accessor}}
           </code>
+          <font class="has-text-grey is-size-8" style="margin-left:10px;">
+            {{#if (eq backend.description "")}}
+              {{#if (eq "ad" backend.type)}}Active Directory credentials & service accounts.{{/if}}
```

Comment:
We should store these strings somewhere centralized (such as `options-for-backend`) and then get them here instead of doing all these if statements

Reply:
Save each string as `description` key to its corresponding engine type in `SECRET_BACKENDS` and then use the helper like 
```hbs
{{#unless backend.description}}
  {{get (options-for-backend backend.type) "description"}}
{{else}} ...
```